### PR TITLE
498 Add logger_handler to LrScheduleHandler

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes https://github.com/Project-MONAI/tutorials/discussions/498.

### Description
This PR added `logger_handler` to the `LrScheduleHandler`, same as other MONAI components, like:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/handlers/stats_handler.py#L57
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/utility/array.py#L561

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/handlers/lr_schedule_handler.py [hunk 1]
```diff
@@ -47,6 +48,9 @@ def __init__(
                 `True` is epoch level, `False` is iteration level.
             step_transform: a callable that is used to transform the information from `engine`
                 to expected input data of lr_scheduler.step() function if necessary.
+            logger_handler: if `print_lr` is True, add additional handler to log the learning rate: save to file, etc.
+                all the existing python logging handlers: https://docs.python.org/3/library/logging.handlers.html.
+                the handler should have a logging level of at least `INFO`.
```
Comment (wyli) [inline monai/handlers/lr_schedule_handler.py:53]: is this line 53 copied here by mistake?

Comment (Nic-Ma) [reply monai/handlers/lr_schedule_handler.py:53]: Hi @wyli ,

It's not a mistake, the `logger_handler` should take level > INFO as we use `logger.info` to save:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/handlers/lr_schedule_handler.py#L85

Thanks.

File: tests/test_handler_stats.py [hunk 2]
```diff
@@ -136,18 +136,19 @@ def _train_func(engine, batch):
             # set up testing handler
             stats_handler = StatsHandler(name=key_to_handler, tag_name=key_to_print, logger_handler=handler)
             stats_handler.attach(engine)
+            stats_handler.logger.setLevel(logging.INFO)
 
             engine.run(range(3), max_epochs=2)
             handler.close()
             stats_handler.logger.removeHandler(handler)
             with open(filename) as f:
                 output_str = f.read()
```
Comment (wyli) [inline tests/test_handler_stats.py:145]: What was the issue with this test? I remember previously it worked

Comment (Nic-Ma) [reply tests/test_handler_stats.py:145]: Hi @wyli ,

Sorry I forgot to describe the issue here.
The previous test code actually didn't run into the `assert`:
```py
for idx, line in enumerate(output_str.split("\n")):
    if grep.match(line):  # ---------------> here is FALSE, so following logic didn't run at all and passed
        if idx in [1, 2, 3, 6, 7, 8]:
            self.assertTrue(has_key_word.match(line))
```

Thanks.

Comment (wyli) [reply tests/test_handler_stats.py:145]: Right, looks like we have the same issue for the other test cases https://app.codecov.io/gh/Project-MONAI/MONAI/blob/dev/tests/test_handler_stats.py

Comment (Nic-Ma) [reply tests/test_handler_stats.py:145]: Hi @wyli ,

Nice catch by the tool! Let me update them all.

Thanks.

**Review by wyli — APPROVED** [review]
Thanks, it looks good to me.

File: tests/test_handler_stats.py (hunk 2 above)
Comment (Nic-Ma) [reply tests/test_handler_stats.py:148]: Updated all the wrong test cases in the latest commit.

Thanks.

**Pull request merged**